### ISSUE TITLE
API-39:  Epic for Delivering MicroProfile 2.2

### ISSUE THREAD

**Issue by kwsutter** [OpenLiberty/open-liberty#6722]
This epic will be used to cover the platform MicroProfile 2.2:
https://bigblue.aha.io/features/API-39

MicroProfile 2.2 is based on Java EE 8 technologies. The updated component features for this release are tracked via their own features and #epics.

```
Open API 1.1, https://bigblue.aha.io/features/WC-22, https://github.com/OpenLiberty/open-liberty/issues/6154     
Open Tracing 1.3, https://bigblue.aha.io/features/LG-51, https://github.com/OpenLiberty/open-liberty/issues/6338 
Rest Client 1.2, https://bigblue.aha.io/features/API-35, https://github.com/OpenLiberty/open-liberty/issues/5941
Fault Tolerance 2.0, https://bigblue.aha.io/features/API-22, https://github.com/OpenLiberty/open-liberty/issues/4080
```

**Comment by marikaj123** [comment]
@kwsutter Do you have English files for this feature that requires translation?

**Comment by marikaj123** [comment]
@kwsutter - Does the feature has files that require translation?

**Comment by kwsutter** [comment]
> @kwsutter - Does the feature has files that require translation?

No translation required, @marikaj123.  Thanks.

**Comment by kwsutter** [comment]
WAD link:  https://ibm.box.com/s/mgmnahzfvqzexay0a7twpgo74gaw6bl0  (No WAD Review Approved by Alasdair.  I had still created the WAD document for reference purposes.)

**Comment by kwsutter** [comment]
ID Issue:  https://github.ibm.com/was-liberty/liberty-docs/issues/1142

**Comment by kwsutter** [comment]
Feature Test Summary Issue:  https://github.com/OpenLiberty/open-liberty/issues/6899

**Comment by kwsutter** [comment]
Liberty microProfile-2.2 feature definition PR:  https://github.com/OpenLiberty/open-liberty/pull/6860

**Comment by kwsutter** [comment]
Packaging updates required for MP 2.2:  https://github.ibm.com/was-liberty/WS-CD-Open/issues/15963

**Comment by atosak** [comment]
Feature readiness template "
https://github.com/OpenLiberty/open-liberty/issues/6911

**Comment by kwsutter** [comment]
Per my discussion with @arthurdm and the discussion in the [previous issue created for MP 2.1](https://github.com/OpenLiberty/ci.docker/issues/44), the creation of the OL specific docker images should be automated based off the changes that will go in via the [issue #15963 mentioned earlier](https://github.ibm.com/was-liberty/WS-CD-Open/issues/15963).

For the WebSphere Liberty specific images, I have created the following Issue for tracking purposes:  https://github.ibm.com/WASCloudPrivate/roadmap-cloudpaks/issues/453

**Comment by atosak** [comment]
I had a quick call with Kevin and confirm we need no accessibility, servicibility and globalization for this feature. @marikaj123 , @gscottj , @donbourne , can you please add focal point approval tags. Thank you.

**Comment by marikaj123** [comment]
@atosak - Thanks, I added globalization approval.

**Comment by sabolo** [comment]
Can you please confirm this is just a wrapper epic and the testing of each technology is covered by the individual features referenced in the description?  Assuming so, the FAT approval of this feature will be completed once each of the referenced features are complete.

**Comment by sabolo** [comment]
Should each of the references technologies also be added to this Epic as child issues?

**Comment by kwsutter** [comment]
@sabolo, yes, this is a wrapper epic and the testing of each technology is covered by the individual features.  As you have noticed, I have referenced both the AHA Feature and Github issues for each of the individual features.  I do not know how to link them as child issues.  If someone can point me at directions for doing so, I'd be happy to do that.  Thanks!

**Comment by sabolo** [comment]
At present the fault tolerance Epic that this feature depends on is being held out due to 2 TCK tests being disabled from the automated runs.  As per that Epic, I believe the product risk is minimal here as we have run tests previously as well as some manual testing.  I would recommend we still allow this feature to go onto the release train.  The FAT approval will be withheld until the fault tolerance Epic is complete however this is purely to ensure we get the technical debt completed as a priority.
